samtranslator/open_api/open_api.py

```diff
@@ -37,10 +37,10 @@ def __init__(self, doc):
         modifications on this copy.
 
         :param dict doc: OpenApi document as a dictionary
-        :raises ValueError: If the input OpenApi document does not meet the basic OpenApi requirements.
+        :raises InvalidDocumentException: If the input OpenApi document does not meet the basic OpenApi requirements.
```

Comment:
Should this be InvalidDocument or InvalidResource?

Reply:
We can throw `InvalidResource` but we need logical id of the resource.

Reply:
Got it. That gets me thinking but nothing specific for this PR.